src/test/signalk/SKNMEA2000ParserTest.cpp

```diff
@@ -83,5 +83,35 @@ TEST_CASE("SKNMEA2000Parser: Basic tests") {
     CHECK( update.getEnvironmentDepthBelowKeel() == 40.8 );
     CHECK( update.getEnvironmentDepthTransducerToKeel() == 1.2 );
   }
-}
 
+  SECTION("127250: VESSEL True HEADING RAPID ") {
+    SetN2kTrueHeading(msg, 0, 180);
+    const SKUpdate &update = p.parse(SKSourceInputNMEA2000, msg, SKTime(0));
+    CHECK( update.getSize() == 1);
+    CHECK( update.getNavigationHeadingTrue() == 180 );
+  }
+
+  SECTION("127250: VESSEL Magnetic HEADING RAPID ") {
+    SetN2kMagneticHeading(msg, 0, 181, -2, 3);
+    const SKUpdate &update = p.parse(SKSourceInputNMEA2000, msg, SKTime(0));
+    CHECK( update.getSize() == 2);
+    CHECK( update.getNavigationHeadingMagnetic() == 181 );
+    // CHECK( update.getNavigationMagneticDeviation() == -2 );
+    CHECK( update.getNavigationMagneticVariation() == 3 );
+  }
+
+  SECTION("127245: Rudder Angle") {
+    SetN2kRudder(msg,-3.5, 0);
```

Comment:
You need to convert in radians here. Otherwise -3.5 is not what you think it is.